web/modules/custom/usa_orphaned_entities/src/Form/OrphanedEntitiesSettings.php

```diff
@@ -35,8 +35,10 @@ public function getFormId(): string {
 
   /**
    * {@inheritdoc}
+   *
+   * @param array<mixed> $form
+   * @return array<mixed>
```

Comment:
same as on the other array $forms can be narrowed to `array<string, mixed>` as args and return types